### DISCUSSION TITLE
Not syncing and I have no idea how to debug

### DISCUSSION THREAD

**Discussion by ddt3** [vrtmrz/obsidian-livesync#333] [Q&A]
I am using Obsidian 1.4.16
LiveSync 0.20.7

Yesterday I imported all my evernote into obsidian. 
Livesync appeared to sync the first folder but after that it just stops. Is there a way for me to find out which note is causing this? Anything else I can do?

This is what the log shows:
```
1/21/2023, 7:27:26 AM->Replication activated
11/21/2023, 7:27:27 AM->HTTP:POST (4608480) to:/_bulk_docs -> failed
11/21/2023, 7:27:27 AM->TypeError:Failed to fetch
11/21/2023, 7:27:27 AM->Replication paused
11/21/2023, 7:27:27 AM->Replication activated
11/21/2023, 7:27:29 AM->HTTP:POST (4608382) to:/_bulk_docs -> failed
11/21/2023, 7:27:29 AM->TypeError:Failed to fetch
11/21/2023, 7:27:29 AM->Replication paused
11/21/2023, 7:27:29 AM->Replication activated
11/21/2023, 7:27:30 AM->HTTP:POST (4608791) to:/_bulk_docs -> failed
11/21/2023, 7:27:30 AM->TypeError:Failed to fetch
11/21/2023, 7:27:30 AM->Replication paused
11/21/2023, 7:27:30 AM->Replication activated
11/21/2023, 7:27:32 AM->HTTP:POST (4608281) to:/_bulk_docs -> failed
11/21/2023, 7:27:32 AM->TypeError:Failed to fetch
11/21/2023, 7:27:32 AM->Replication paused
11/21/2023, 7:27:32 AM->Replication activated
```

**Comment by dlphnkck** [comment]
Hi. I'm also new to the LiveSync plugin, but I think there is not enough information.
If you don't mind, would you fill in the following template information?

[obsidian\-livesync/\.github/ISSUE\_TEMPLATE/issue\-report\.md at 9747c26d50cda47169f2a7682d384e87e52c19c4 · vrtmrz/obsidian\-livesync](https://github.com/vrtmrz/obsidian-livesync/blob/9747c26d50cda47169f2a7682d384e87e52c19c4/.github/ISSUE_TEMPLATE/issue-report.md)
